JetStreamCompose/gradle/wrapper/gradle-wrapper.properties

```
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.9-bin.zip
```

newer version is available
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.12-bin.zip
```